ceph-dev-build/build/build_rpm

```diff
@@ -84,7 +84,7 @@ fi
 
 # XXX perhaps use job parameters instead of literals; then
 # later stages can also use them to compare etc.
-if [[ "$CI_CONTAINER" == true && $DISTRO == "centos" && $RELEASE == "8" ]] ; then
+if [[ "$CI_CONTAINER" == true && $DISTRO == "centos" && "$RELEASE" == 9 ]] ; then
```

Comment:
@dmick does this change only apply to main and squid?

Reply:
It applies to whatever requests a c9 build.  I'm assuming there's a correlation between requesting centos9 and wanting a centos9 container.  I'd like to avoid more ceph-release restrictions in various and sundry places if possible.  I think this is a good assumption, though.

Reply:
Changed to build containers for c8 AND c9.